src/components/signUp/signUpResponse.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Homepage from './../homepage/homepage';
+const SignUpResponse  = ({response})=>{
+
+    console.log(`Response in other file ${response.status}`);
+    
+    const status = response.status;
+
+    if(status === 200 || status === 201){
+        return(<Homepage></Homepage>);
+    }else if(status === 400){
+        return(<h1>Bad Request</h1>)
```

Comment:
Render the server error message here.

Reply:
yeah chandan but as i didn't know which parameters are coming from server with status code so i wrote that message but we can change that sure